app/src/main/res/layout/activity_enter_mnemonic.xml

```diff
@@ -67,6 +53,21 @@
                 android:maxLines="1" />
         </com.google.android.material.textfield.TextInputLayout>
 
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/welcomeTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="24dp"
+            android:layout_marginEnd="16dp"
+            android:text="Create wallet"
```

Comment:
Hard coded string? `Create wallet`? Or is it some "hint"?

Reply:
This screen is old and will be gone/redone